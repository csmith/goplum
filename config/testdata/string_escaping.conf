defaults {
    test = "abc\r\n\tdef\\ghi\\\"jkl\"mno"
    invalid = "\x"
}
