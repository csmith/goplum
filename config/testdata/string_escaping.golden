[
  {
    "Class": 17,
    "Line": 1,
    "Column": 0,
    "Value": "defaults"
  },
  {
    "Class": 2,
    "Line": 1,
    "Column": 9,
    "Value": 123
  },
  {
    "Class": 4,
    "Line": 2,
    "Column": 4,
    "Value": "test"
  },
  {
    "Class": 5,
    "Line": 2,
    "Column": 9,
    "Value": 61
  },
  {
    "Class": 6,
    "Line": 2,
    "Column": 11,
    "Value": "abc\r\n\tdef\\ghi\\\"jkl\"mno"
  },
  {
    "Class": 4,
    "Line": 3,
    "Column": 4,
    "Value": "invalid"
  },
  {
    "Class": 5,
    "Line": 3,
    "Column": 12,
    "Value": 61
  },
  {
    "Class": 0,
    "Line": 3,
    "Column": 14,
    "Value": "invalid character escape x"
  }
]